client-js/CONTRIBUTING.md

```diff
@@ -2,17 +2,29 @@
 This module is a JS library that communicates with Spine Web server. It’s a facade for sending 
 domain commands, querying data and subscribing to entity states.  
 
-The latest published version can be found on [NPM](https://www.npmjs.com/package/spine-web-client).
+The latest published version can be found on [NPM](https://www.npmjs.com/package/spine-web).
 
 ## Usage
 
 To use the library install following modules:
 
 ```
-npm i spine-web-client --save
```

Comment:
It's interesting. While reviewing the related `base` changeset I thought we were using `spine-web-client` as an NPM artifact name. However, now I realize we were not. And I like `spine-web` better.

So let's discuss my `base` comment if you need.